prusti-interface/src/specs/mod.rs

```diff
@@ -109,34 +109,13 @@ fn reconstruct_typed_assertion<'tcx>(
     assertion.to_typed(typed_expressions, tcx)
 }
 
-/// Check if `prusti::spec_only` is among the attributes.
-fn has_spec_only_attr(attrs: &[ast::Attribute]) -> bool {
-    attrs.iter().any(|attr| match &attr.kind {
-        ast::AttrKind::Normal(ast::AttrItem {
-            path: ast::Path { span: _, segments, tokens: _ },
-            args: ast::MacArgs::Empty,
-            tokens: _,
-        }) => {
-            segments.len() == 2
-                && segments[0]
-                    .ident
-                    .name
-                    .with(|attr_name| attr_name == "prusti")
-                && segments[1]
-                    .ident
-                    .name
-                    .with(|attr_name| attr_name == "spec_only")
-        }
-        _ => false,
-    })
-}
-
-/// Check if `prusti::extern_spec` is among the attributes.
-fn has_extern_spec_attr(attrs: &[ast::Attribute]) -> bool {
+/// Check if `prusti::<name>` is among the attributes.
+fn has_prusti_attr(attrs: &[ast::Attribute], name: &str) -> bool {
```

Comment:
Maybe we should extract this change into `master`. We also do something very similar in https://github.com/viperproject/prusti-dev/pull/176/files#diff-1fb946419baeafc95d895e440b15e66eR247

Reply:
I agree!

Reply:
#180